falcon/ms_io/mzml_io.py

```diff
@@ -50,21 +52,30 @@ def _parse_spectrum(spectrum_dict: Dict) -> sus.MsmsSpectrum:
     MsmsSpectrum
         The parsed cluster.
     """
-    spectrum_id = spectrum_dict['id']
-    mz_array = spectrum_dict['m/z array']
-    intensity_array = spectrum_dict['intensity array']
-    retention_time = (spectrum_dict['scanList']['scan'][0]
-                      .get('scan start time', -1))
+    # print(spectrum_dict)
+    spectrum_id = spectrum_dict["id"]
+    mz_array = spectrum_dict["m/z array"]
+    intensity_array = spectrum_dict["intensity array"]
+    retention_time = spectrum_dict["scanList"]["scan"][0].get(
+        "scan start time", -1
+    )
 
-    precursor = spectrum_dict['precursorList']['precursor'][0]
-    precursor_ion = precursor['selectedIonList']['selectedIon'][0]
-    precursor_mz = precursor_ion['selected ion m/z']
-    if 'charge state' in precursor_ion:
-        precursor_charge = int(precursor_ion['charge state'])
-    elif 'possible charge state' in precursor_ion:
-        precursor_charge = int(precursor_ion['possible charge state'])
+    precursor = spectrum_dict["precursorList"]["precursor"][0]
+    precursor_ion = precursor["selectedIonList"]["selectedIon"][0]
+    precursor_mz = precursor_ion["selected ion m/z"]
+    if "charge state" in precursor_ion:
+        precursor_charge = int(precursor_ion["charge state"])
+    elif "possible charge state" in precursor_ion:
+        precursor_charge = int(precursor_ion["possible charge state"])
     else:
-        raise ValueError('Unknown precursor charge')
+        precursor_charge = -1
```

Comment:
**todo:** Same for default precursor charge.

Reply:
Done